analyze.sh

```diff
@@ -14,7 +14,7 @@ if [ -d "$ARTIFACT_DIR" ]; then
   cp public/dist/report.html "${ARTIFACT_DIR}"
 fi
 
-MAX_BYTES=3827302 # ~3.65 MiB
+MAX_BYTES=3879731 # ~3.7 MiB
```

Comment:
Necessary since we're pulling in a new dep to vendor main bundle.

Reply:
yeah... we are getting bigger and bigger :(